packages/terra-alert/src/terra-dev-site/doc/example/ActionAndDismissibleExample.jsx

```diff
@@ -1,7 +1,11 @@
 import React from 'react';
 import Button from 'terra-button';
+import classNames from 'classnames/bind';
 // eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
-import Alert from 'terra-alert/lib/Alert';
+import Alert from 'terra-alert';
```

Comment:
For hot reloading to work I think this needs to be reverted back to import from `'terra-alert/lib/Alert'`.

Reply:
Fixed here https://github.com/cerner/terra-core/pull/2407/commits/016e1a078d9ba50f5cf9c9d7c00f95560540025f.